docs/src/fts.md

```diff
@@ -14,63 +16,112 @@ pip install tantivy==0.20.1
 
 ## Example
 
-Consider that we have a LanceDB table named `my_table`, whose string column `text` we want to index and query via keyword search.
+Consider that we have a LanceDB table named `my_table`, whose string column `text` we want to index and query via keyword search, the FTS index must be created before you can search via keywords.
 
-```python
-import lancedb
-
-uri = "data/sample-lancedb"
-db = lancedb.connect(uri)
-
-table = db.create_table(
-    "my_table",
-    data=[
-        {"vector": [3.1, 4.1], "text": "Frodo was a happy puppy"},
-        {"vector": [5.9, 26.5], "text": "There are several kittens playing"},
-    ],
-)
-```
+=== "Python"
 
-## Create FTS index on single column
+    ```python
+    import lancedb
 
-The FTS index must be created before you can search via keywords.
+    uri = "data/sample-lancedb"
+    db = lancedb.connect(uri)
 
-```python
-table.create_fts_index("text")
-```
+    table = db.create_table(
+        "my_table",
+        data=[
+            {"vector": [3.1, 4.1], "text": "Frodo was a happy puppy"},
+            {"vector": [5.9, 26.5], "text": "There are several kittens playing"},
+        ],
+    )
 
-To search an FTS index via keywords, LanceDB's `table.search` accepts a string as input:
+    # passing `use_tantivy=False` to use lance FTS index
```

Comment:
do we have a default value for `use_tantivy`?

Lets show that in the code as well, i.e.,

```
# Create native FTS index
table.create_fts_index("text")
```

Reply:
Modified